openmdao/core/driver.py

```diff
@@ -8,14 +8,34 @@
 
 from openmdao.recorders.recording_manager import RecordingManager
 from openmdao.recorders.recording_iteration_stack import Recording
-from openmdao.utils.record_util import create_local_meta
+from openmdao.utils.record_util import create_local_meta, check_path
+from openmdao.utils.mpi import MPI
 from openmdao.utils.options_dictionary import OptionsDictionary
 
 
 class Driver(object):
     """
     Top-level container for the systems and drivers.
 
+    Options
+    -------
+    options['record_metadata'] :  bool(True)
+        Tells recorder whether to record variable attribute metadata.
+    options['record_desvars'] :  bool(True)
+        Tells recorder whether to record the desvars of the Driver.
+    options['record_responses'] :  bool(False)
+        Tells recorder whether to record the responses of the Driver.
+    options['record_objectives'] :  bool(False)
+        Tells recorder whether to record the objectives of the Driver.
+    options['record_constraints'] :  bool(False)
+        Tells recorder whether to record the constraints of the Driver.
```

Comment:
probably need to emphasize that these are for recording in the descriptions, since they will show up in the options table in every driver feature doc.

Reply:
Can you explain more ? I am struggling to see how I can say more than I already did. The descriptions use some variant of "record" twice. 

Maybe I am missing something ?

Thanks for taking a look at the PR! I know there were a lot of changes

Reply:
Never mind, I think I was more confused about these initially.